src/Components/WebAssembly/Build/src/targets/Blazor.MonoRuntime.targets

```diff
@@ -75,6 +75,13 @@
     <MakeDir Directories="$(_BlazorIntermediateOutputPath)" />
   </Target>
 
+  <Import
+    Project="$(_BlazorSatelliteAssemblyStashFile)"
+    Condition="Exists('$(_BlazorSatelliteAssemblyStashFile)')" />
```

Comment:
I haven't yet understood this part. Why do we `<Import>` the stash file rather than use `<BlazorReadSatelliteAssemblyFile>` to read it?

The `BlazorReadSatelliteAssemblyFile` part I'm pretty comfortable. That's the same kind of "store a manifest file" pattern we use in loads of places. But the technique of "write out an actual `.targets` file and then import it" is quite novel.

Presumably this is essential for some reason otherwise you wouldn't be doing it, but if so, could you clarify why it's not sufficient to write the data in whatever custom format we want (e.g., `.xml` file) and then use `BlazorReadSatelliteAssemblyFile` to read it back and generate a corresponding ItemGroup?